Sources/Commands/PackageCommands/ResetCommands.swift

```diff
@@ -45,8 +45,8 @@ extension SwiftPackageCommand {
         @OptionGroup(visibility: .hidden)
         var globalOptions: GlobalOptions
 
-        func run(_ swiftCommandState: SwiftCommandState) throws {
-            try swiftCommandState.getActiveWorkspace().reset(observabilityScope: swiftCommandState.observabilityScope)
+        func run(_ swiftCommandState: SwiftCommandState) async throws {
```

Comment:
Is there a specific reason `SwiftPackageCommand.Reset` shouldn't conform to `AsyncSwiftCommand` now?